src/cript/api/api.py

```diff
@@ -683,7 +684,18 @@ def _internal_save(self, node, save_values: Optional[_InternalSaveValues] = None
             # If we get an error we may be able to fix, we to handle this extra and save the bad node first.
             # Errors with this code, may be fixable
             if response["code"] in (400, 409):
-                returned_save_values = _fix_node_save(self, node, response, save_values)
+                try:
+                    returned_save_values = _fix_node_save(self, node, response, save_values)
+                except CRIPTAPISaveError as exc:
+                    # If the previous error was a duplicated name issue
+                    if "duplicate item [{'name':" in str(response["error"]):
+                        # And (second condition) the request failed bc of the now suppressed name
+                        if "'name' is a required property" in exc.api_response:
```

Comment:
I wonder if this would be cleaner if we remove the nested if and combine them into an if statement with `and`

Reply:
fair point, agreed!